tinyagentos/routes/hub.py

```
@@ -109,3 +147,356 @@ async def put_own_profile(
+        "blocks": await _peers(relationships.REL_BLOCK),
+        "mutes": await _peers(relationships.REL_MUTE),
+        "requests_out": await _peers(relationships.REL_REQUEST_OUT),
+        "requests_in": await _peers(relationships.REL_REQUEST_IN),
+        "requests_declined": await _peers(relationships.REL_REQUEST_DECLINED),
+    }
+
+
+@router.get("/api/hub/presence")
+async def lookup_presence(
+    request: Request,
+    user: CurrentUser = Depends(current_user),
```

Ruff (0.15.20)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)